### PR TITLE
[UT] Add some DDL test cases

### PR DESCRIPTION
**Pull request by sduzh**
## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [x] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

**Review by Astralidea — DISMISSED** [review]

**Review by gengjun-git — DISMISSED** [review]

File: test/sql/test_alter_table/T/test_alter_table_abnormal [hunk 1]
```diff
@@ -0,0 +1,98 @@
+-- name: test_alter_pk_table_abnormal
+
+CREATE TABLE t0(k0 BIGINT, k1 DATETIME, v0 INT, v1 VARCHAR(100))
+ primary key(k0, k1)
+ distributed by hash(k0)
+ buckets 1
+ order by (v0)
+ properties('replication_num'='1');
+
```
Comment (imay) [inline test/sql/test_alter_table/T/test_alter_table_abnormal:9]: Why not inserting some data into this table and add some select statements to test if the column is added/dropped correctly?

Comment (sduzh) [reply test/sql/test_alter_table/T/test_alter_table_abnormal:9]: I have added an `DESC <table>` statement in the tests to check the table structure.

Comment (imay) [reply test/sql/test_alter_table/T/test_alter_table_abnormal:9]: Got it, how about inserting some data?

Comment (sduzh) [reply test/sql/test_alter_table/T/test_alter_table_abnormal:9]: Updated

**Review by gengjun-git — APPROVED** [review]

**Review by Astralidea — APPROVED** [review]

**Pull request merged**